shared/torngit/bitbucket.py

```diff
@@ -212,20 +212,19 @@ async def delete_webhook(self, hookid, token=None):
 
     async def get_is_admin(self, user, token=None):
         user_uuid = "{" + user["service_id"] + "}"
-        query = f'workspace.uuid="{{{self.data["owner"]["service_id"]}}}" AND permission="owner" AND user.uuid="{user_uuid}"'
+        workspace_uuid = f'{{{self.data["owner"]["service_id"]}}}'
```

Comment:
Nit: thoughts on making these consistent? They both add `{ }` around the variable, so it would be more consistent if both used f strings or concatenated `{ }`.